.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.

```

Comment:
I'm looking at the templates in React Native and they use HTML comments for text that you don't want to see in the issue:

https://github.com/facebook/react-native/edit/master/.github/ISSUE_TEMPLATE.md

I would do the same since you probably don't want to have this text on the issue.